src/coldcard.ts

```diff
@@ -410,15 +392,16 @@ export class ColdcardExportExtendedPublicKey extends ColdcardMultisigSettingsFil
  *
  */
 export class ColdcardSignMultisigTransaction extends ColdcardInteraction {
-  /**
-   *
-   * @param {object} options - options argument
-   * @param {string} options.network - bitcoin network
-   * @param {array<object>} options.inputs - inputs for the transaction
-   * @param {array<object>} options.outputs - outputs for the transaction
-   * @param {array<string>} options.bip32Paths - BIP32 paths
-   * @param {object} [options.psbt] - PSBT of the transaction to sign, include it or we will generate it
-   */
+  network: string;
+
+  psbt: any;
+
+  inputs: any[];
+
+  outputs: any[];
+
+  bip32Paths: string[];
```

Comment:
I think when we can avoid `any` we should. Maybe this caused downstream failures that expanded the scope too much so feel free to reject if you already tried something like this. 

```suggestion
  network: BitcoinNetwork;

  psbt: Record<string, unknown>;

  inputs: Record<string, unknown>[];

  outputs: Record<string, unknown>[];

  bip32Paths: string[];
```

Reply:
I think these were added as `any` for the exact reason you mention. This was a series of timeboxed changes and the minimum was to get everything into typescript files and building, but not necessarily to fix all the typing. If all the types must be solid, then it might need to wait for another timebox.

Reply:
Yeah that’s fair enough. Can I request an update maybe to the top of the readme regarding types and indicate some of these examples I highlight in my comments? It could be a good first task for someone looking to get their feet wet (as we recently had in caravan), so nice to give some explicit direction, also just for keeping track of this not being the state we want long term. 

I didn’t check but are we at least warning for `any` and we can eventually turn to an error?

Reply:
Readme note added in 8a7d8b3ff59d054336ed85b21940f79453a7f606

There is no `any` warning. It will error if tsconfig is updated to have `noExplicitAny`.

Reply:
I think we could do it with an eslint rule:

```
module.exports = {
  "rules": {
    "@typescript-eslint/no-explicit-any": "warn"
  }
};
```

but I don't feel super strongly about it. was just a thought about how to get us to eventually clean these up without blocking this update.